channels/generic/websockets.py

```diff
@@ -249,27 +266,28 @@ class WebsocketMultiplexer(object):
     stream = None
     reply_channel = None
 
-    def __init__(self, stream, reply_channel):
+    def __init__(self, stream, reply_channel, json_encoder=_default_json_encoder):
```

Comment:
I'd rather this was a None value, and it directly fell back to json.dumps rather than a global variable